tests/robot-tests/tests/libs/no_javascript.py

```diff
@@ -0,0 +1,33 @@
+from robot.libraries.BuiltIn import BuiltIn
+sl = BuiltIn().get_library_instance('SeleniumLibrary')
+import os
+import requests
+from bs4 import BeautifulSoup
+
+def user_checks_page_accordions_work_with_no_javascript(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, "html.parser")
+
+    first_accordion = soup.select_one("[data-testid='accordionSection']")
+    if not first_accordion:
+        raise AssertionError('No accordionSection found!')
+
+    first_heading = first_accordion.select_one("#publications-1-heading")
+    if not first_heading:
+        raise AssertionError('accordionSection first heading wasn\'t found!')
+    if not 'aria-expanded' in first_heading.attrs:
+        raise AssertionError('First accordion doesn\'t contain "aria-expanded" attribute')
+
```

Comment:
There should be some additional for content within the accordion

Reply:
Discussed this with Nick. I'm going to skip checking the content for now. The content does appear without JS, and these tests  are just a quick fix to help me avoid having to manually test with no JS all the time.

In the long run, we hope to get Chrome headless working with no JS, and then we can do a much more thorough and complete no JS automated test - but that requires either Selenium 4 or some other way to access the Chrome DevTools Protocol.